src/App.jsx

```diff
@@ -14,33 +16,93 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABET = 'alphabet';
+const SORT_LENGHT = 'lenght';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+const getPreparedGoods = (sortMethod, isReverse) => {
+  const gooods = [...goodsFromServer];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  gooods.sort((a, b) => {
+    switch (sortMethod) {
+      case SORT_ALPHABET:
+        return a.localeCompare(b);
+      case SORT_LENGHT:
+        return a.length - b.length;
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (isReverse) {
+    gooods.reverse();
+  }
+
+  return gooods;
+};
+
+export const App = () => {
+  const [goodSort, setGoodSort] = useState('');
+  const [isSortReverse, setSortIsReverse] = useState(false);
```

Comment:
```suggestion
  const [goodsSortType, setGoodsSortType] = useState('');
  const [isReversed, setIsReversed] = useState(false);
```
Very strange naming, how can sort be reversed? Also, pay attention to the name convention for `useState`
https://react.dev/reference/react/useState
![image](https://github.com/mate-academy/react_list-of-goods-js/assets/63243136/f50cf683-bbec-42b1-becb-75c9c9dd1a14)